source/preprocessor.tex

```diff
@@ -592,18 +592,15 @@
 An identifier currently defined as an
 \indextext{object-like macro|see{macro, object-like}}%
 \indextext{macro!object-like}%
-\grammarterm{object-like}
-macro may be redefined by another
+object-like macro may be redefined by another
```

Comment:
Please add a parenthetical "(see below)" to each of these uses preceding the definition. Please also convert the `\grammarterm`s on lines 668 and 692 to `\defn`s.